README.md

```diff
@@ -256,10 +263,21 @@ If you like this project please __Star us on GitHub — it helps!__ :star: :star
 
 Please visit the [PcapPlusPlus web-site](https://pcapplusplus.github.io/community) to learn more.
 
+[![GitHub issues](https://img.shields.io/github/issues/seladb/PcapPlusPlus?style=flat&label=Issues&logo=github)](https://github.com/seladb/PcapPlusPlus/issues/new/choose)
+[![Google Groups](https://img.shields.io/badge/Google%20Groups-%234285F4.svg?&style=flat&logo=google&logoColor=white)](https://groups.google.com/d/forum/pcapplusplus-support)
+[![Stack Overflow](https://img.shields.io/badge/Stack%20Overflow-%23F58025.svg?&style=flat&logo=stackoverflow&logoColor=white)](https://stackoverflow.com/questions/tagged/pcapplusplus)
+[![E-Mail](https://img.shields.io/badge/E--Mail-%230078d4.svg?&style=flat&logo=minutemailer&logoColor=white)](mailto:pcapplusplus@gmail.com)
+[![Twitter Follow](https://img.shields.io/twitter/follow/seladb?label=Follow%20PcapPlusPlus&style=social)](https://twitter.com/intent/follow?screen_name=seladb)

```

Comment:
These badges kind of repeat the same information that exists in the previous paragraph, I don't know if there's a nice way to combine them?

Reply:
Now when I think about it - maybe we can try to turn it into a table like in the web-site?
https://pcapplusplus.github.io/community#contact-us

I'm not sure it'll look nicer, but it's worth a try. What do you think?

Reply:
I'm not sure about the background of table 🤔

![image](https://user-images.githubusercontent.com/64282645/194119708-7858bafe-b692-45cb-ac1e-ead992b55d2b.png)

Maybe we should leave it as is 😁

Reply:
yeah, maybe we can leave it as is :)